rsc/src/main/java/com/cloudera/livy/rsc/RSCConf.java

```diff
@@ -72,7 +72,7 @@
     SASL_QOP("rpc.sasl.qop", null),
 
     TEST_STUCK_END_SESSION("test.do_not_use.stuck_end_session", false),
-    TEST_PING_JOB_ERROR("test.do_not_use.ping_job_error", false);
+    TEST_STUCK_START_SESSION("test.do_not_use.stuck_start_session", false);
```

Comment:
It's blocking the entire driver to start. Mind if we rename it to TEST_STUCK_START_DRIVER?
LGTM otherwise.